src/App.js

```diff
@@ -107,7 +107,7 @@ const App = ({ locale, location, window, dependencies: { appSessionRef, sessionM
                   location.hash.length && process.env.REACT_APP_ROUTER !== 'hash' ? (
                     <Redirect to={location.hash.replace('#', '')} />
                   ) : (
-                    <SafeRedirect to="/Campaigns/Draft" />
+                    <SafeRedirect to="/Dashboard" />
```

Comment:
This is wrong, it will affect all doppler. Please remove it.

Reply:
Corrected!